csep/utils/comcat.py

```diff
@@ -313,7 +313,8 @@ def _search(**newargs):
 
     except URLError as URLe:
         # Fails to verify SSL certificate, when there is a hostname mismatch
-        if isinstance(URLe.reason, ssl.SSLCertVerificationError) and URLe.reason.verify_code == 62:
+        if (isinstance(URLe.reason, ssl.SSLCertVerificationError) and URLe.reason.verify_code == 62) \
+                or (isinstance(URLe.reason, ssl.SSLError) and URLe.reason.errno == 5):
```

Comment:
Added additional reason and error code to make this compatible with Python 3.8-3.11.